### DISCUSSION TITLE
Migration from NPM using MariaDB possible?

### DISCUSSION THREAD

**Discussion by sym0nd0** [ZoeyVid/NPMplus#944] [Q&A] [answered]
Hi all,
I'm going to move over my main server from NPM to NPMplus. I have used MariaDB with NPM and just want to check if it's possible to migrate to NPMplus and retain existing proxy hosts etc?

I'm assuming not and it's not a massive deal if that's the case, but figure I'll ask first in case it can save me some time.

Finally, ZoeyVid I just want to say thanks for your work and continuing effort on this.

**Comment by Zoey2936** [comment, marked as answer]
It should work, but is unsupported, so make a backup first
